vignettes/getting_started.Rmd

```diff
@@ -1,6 +1,15 @@
 ---------------------
 
-author: "Jamaal Green"
+author:
```

Comment:
This doesn't look right, and there are some ways we should fix this in the DESCRIPTION file.

Reply:
Remember to include Liming in here also as an author.